pkgs/tools/misc/hdf5/default.nix

```diff
@@ -91,6 +91,20 @@ stdenv.mkDerivation rec {
     moveToOutput 'bin/h5pcc' "''${!outputDev}"
   '';
 
+  postFixup = ''
+    # Remove reference to /build, which get introduced
```

Comment:
If you would move the comment out of the bash line the hash of the derivation would not change upon just changing the comment.